srcpkgs/libsignal-protocol-c/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'libsignal-protocol-c'
+pkgname=libsignal-protocol-c
+version=2.3.2
+revision=1
+build_style=cmake
+makedepends="check-devel lcov libressl-devel"
```

Comment:
isn't check-devel used only when building tests?

Reply:
Yeah, it should.

Reply:
I would like to integrate the check, if someone has a suggestion, how I do that.

Reply:
I have left this for now.